app/models/event/question.rb

```diff
@@ -93,6 +93,18 @@ def derived?
     derived_from_question_id.present?
   end
 
+  def required?
+    disclosure&.to_sym == :required
```

Comment:
Since you generate the queries with the i18n_enum, this could be

```suggestion
    disclosure&.required?
```

(if I understand it correctly, of course)

The same goes for the next two queries.

Reply:
Ich habe das noch ausprobiert, leider funktioniert das nicht out of the box. Es braucht dafür den [StringInquirer](https://api.rubyonrails.org/classes/ActiveSupport/StringInquirer.html). 

Soll ich das mit dem umsetzen oder lassen wir's bei dieser Implementation?

Reply:
ah, jetzt sehe ich es. Wenn `queries: true` ist, wird direkt eine passende Methode definiert:

```
      possible_values&.each do |value|
        scope value.to_sym, -> { where(attr => value) } if scopes
        define_method(:"#{value}?") { self[attr] == value } if queries
      end
```

Damit wäre deine Query-Method gar nicht mehr nötig...

Reply:
Den StringInquirer würde ich **nicht** einbauen.

Reply:
Ah jetzt ists klar. Danke!